packages/shared-components/utils/chartUtils/chartConfig.jsx

```
@@ -0,0 +1,298 @@
+  if (topPaddingPercentage < minPaddingPercentage) {
+    topPaddingPercentage = minPaddingPercentage;
+  }
+  let bottomPaddingPercentage = (maxPaddingPercentage - Math.log10(min));
+  if (bottomPaddingPercentage < minPaddingPercentage) {
+    bottomPaddingPercentage = minPaddingPercentage;
+  }
+  min -= (min / 100) * bottomPaddingPercentage;
+  max += (max / 100) * topPaddingPercentage;
+  return {
+    min,
+    max,
+  };
+}
+
+export const setYAxisScale = ([primarySeries, secondarySeries, ...otherSeries], xAxis, yAxis) => {
+  // we are using two scales only if there is a siginificative difference in scale
+  const yAxisMax = Math.max.apply(null, reduceSeriesToValues([primarySeries]));
+  let usesTwoScales = false;
+  if (secondarySeries) {
+    const secondaryYAxisMax = Math.max.apply(null, reduceSeriesToValues([secondarySeries]));
+    const maxScaleDifference = 0.7;
+    let scaleDifference = 0;
+    if (yAxisMax > secondaryYAxisMax) {
+      scaleDifference = (Math.log10(yAxisMax) - Math.log10(secondaryYAxisMax));
+    } else {
+      scaleDifference = (Math.log10(secondaryYAxisMax) - Math.log10(yAxisMax));
+    }
+    if (scaleDifference >= maxScaleDifference && !xAxis.categories) {
+      secondarySeries.yAxis = 1;
+      usesTwoScales = true;
+    } else {
+      secondarySeries.yAxis = 0;
+    }
+  }
+  setChartLimits({
+    series: [].concat(primarySeries, secondarySeries, otherSeries).filter(s => typeof s !== 'undefined'),
+    yAxis,
+  }, usesTwoScales);
+}
+
+export const getMarkerFillColor = (color) => {
+  const highchartColor = ReactHighcharts.Highcharts.Color(color);
+  return highchartColor ?
+    highchartColor.brighten(0.1).get('rgba') :
+    color;
+}
+
+export const fadeColor = (color, opacity) => {
+  const highchartColor = ReactHighcharts.Highcharts.Color(color);
+  return highchartColor ?
+    highchartColor.setOpacity(opacity).get('rgba') :
+    color;
+}
+
+export default (tooltipgetter, stacked = false, type = 'areaspline') => {
```

Super small one, let's change `tooltipgetter` into `tooltip`. Highchart is already expecting a method there.
Also given that this is the only one without a default and we might find the need to hide the tolltip for some series, it might be worth setting the default to  something like this `(() => (null))`.